neuland/simulation/generator/R3BNeulandGenerators.h

```diff
@@ -13,7 +13,7 @@ inline auto create_muon_generator(TRandom3& random_gen)
 {
     constexpr auto num_DP = 13;
     constexpr auto PID = 2112;
-    constexpr auto z_pos = 0.;
+    constexpr auto z_pos = 1650.0;
```

Comment:
This value should not be hard coded in the source code. Better make it as a member variable and set it using a setter public member function.